frontends/mit-open/.storybook/preview-head.html

```diff
@@ -0,0 +1,5 @@
+<!--
+      Font files for Adobe Neue Haas Grotesk.
+      WARNING: This is linked to chudzick@mit.edu's Adobe account.
+  -->
+<link rel="stylesheet" href="https://use.typekit.net/lbk1xay.css" />
```

Comment:
How can you tell that it's associated with your email and not with MIT's site license?

Would you feel more comfortable using the same package as www.mit.edu, https://use.typekit.net/uby1wjv.css ?

Reply:
@pdpinch I don't really feel great using https://use.typekit.net/uby1wjv.css either. I think that's a link for an adobe "Web Fonts Project" (like the one I made) but who manages it? What if they change it? Or delete the account and swap to a different one? 

Regarding
> How can you tell that it's associated with your email and not with MIT's site license?
Well, I *think* it's both:
- My Adobe account is via MIT through Touchstone login. So in that sense it's the MIT site license.
- but I believe `lbk1xay` It's associated with me, because I created the project
    <img width="1746" alt="Screenshot 2024-05-13 at 1 03 00 PM" src="https://github.com/mitodl/mit-open/assets/9010790/b9f9c2d9-4249-4d7a-a24b-b8b605f55c5a">

I'm not entirely sure it would disappear if my touchstone account were removed, but it seems reasonable to assume.

You probably found this, but I put related info on https://github.com/mitodl/hq/issues/4205#issuecomment-2103029701

Reply:
@pdpinch Re
> @pdpinch I don't really feel great using https://use.typekit.net/uby1wjv.css (the stylesheet for https://mit.edu) either. I think that's a link for an adobe "Web Fonts Project" (like the one I made) but who manages it? What if they change it? Or delete the account and swap to a different one? 

On the other hand, maybe that link is intended for internal MIT use outside the homepage. I haven't looked for documentation on that (within MIT) but it seems reasonable, and if we could find out more info about that, then that would be a great solution.

Reply:
I'll add—using the "Web Project" fonts stylesheet i created is intended as an interim solution.

Assuming we merge this, I'll open an issue based on https://github.com/mitodl/hq/issues/4205#issuecomment-2103029701 for finding a better alternative.

Reply:
The only technical guidelines I found on https://brand.mit.edu/typography about font inclusion were, more or less, "follow Adobe's procedure via your touchstone account"